packages/theme/src/cli/commands/theme/info.ts

```diff
@@ -0,0 +1,48 @@
+import ThemeCommand from '../../utilities/theme-command.js'
+import {getTheme} from '../../utilities/theme-store.js'
+import {cli, os, output, string} from '@shopify/cli-kit'
+import {version as rubyVersion} from '@shopify/cli-kit/node/ruby'
+import {checkForNewVersion} from '@shopify/cli-kit/node/node-package-manager'
+
+export default class ThemeInfo extends ThemeCommand {
+  static description = 'Print basic information about your theme environment'
+
+  static flags = {
+    ...cli.globalFlags,
+  }
+
+  public async run(): Promise<void> {
+    const sections: [string, string][] = [await this.themeConfigSection(), await this.systemInfoSection()]
+    const message = sections.map((sectionContents) => output.section(...sectionContents)).join('\n\n')
+    output.info(message)
```

Comment:
What about keeping the command's responsibility small (parsing and validating arguments) and delegating the logic to a `services/theme/info.ts` service. It's not a lot of logic, but we'd remain consistent with other commands and we can unit-test that logic outside of the context of the CLI interface.